### DISCUSSION TITLE
friends

### DISCUSSION THREAD

**Discussion by giscus** [water2027/InnerVoice#7] [Announcements]
# friends

blog

https://blog.watering.top/friends

**Comment by Peter267** [comment]
名称：Peter
链接：https://peter267.zeabur.app
头像：https://wmimg.com/i/1169/2024/09/66ec1739c38a0.png
简要描述：无限进步

**Reply by Peter267** [reply]
马上添加您的友链！

**Reply by water2027** [reply]
已经加啦，如果没看到的话可能需要强制刷新一下

**Reply by Peter267** [reply]
感谢。
让我们一起无限进步！

**Reply by water2027** [reply]
不好意思，换域名了，麻烦把链接换成 https://www.w4ter.com         
😊

**Reply by Peter267** [reply]
好久没看，抱歉！
已经更换链接和头像地址

**Comment by Peter267** [comment]
博主您好，修改下友链信息（域名），谢谢！
网站昵称：PeterTech
链接：https://peter267.github.io
原链接：https://peter267.zeabur.app
（如果您需要RSS：https://peter267.github.io/atom.xml）

**Reply by water2027** [reply]
好的，已经改了。可能需要强制刷新才能看到

**Reply by Peter267** [reply]
博主您好，由于本博客（PeterTech，https://peter267.github.io）主要内容方向逐渐变化，麻烦您将博客名称改为“Peter267”，博客类型暂无需改动，感谢！

**Comment by DohhhDo** [comment]
很清爽简洁，现在备案了不太好要友链，倒是可以常到这里看看。
